Sources/CreateAPI/Generator/Templates.swift

```diff
@@ -316,17 +317,17 @@ final class Templates {
                 """
             }
         }
-        
+
         statements += """
-        
+
         default:
             throw DecodingError.dataCorruptedError(in: container, debugDescription: "Failed to initialize `oneOf`")
         }
         """
-        
+
         return """
         \(access)init(from decoder: Decoder) throws {
-            
+
             struct Discriminator: Decodable {
                 let \(discriminator.propertyName): String
             }
```

Comment:
This is what caused the change to generated code @LePips. 

I don't think it's an issue since its just whitespace (better to strip it though) but its worth being mindful that the linter might mistakenly mess some other stuff up.